client_admin/pulp/client/admin/tasks.py

```diff
@@ -162,6 +175,33 @@ def cancel(self, **kwargs):
             else:
                 raise
 
+    def purge(self, **kwargs):
```

Comment:
@ipanova I intentionally kept this method in the base class because it has the future scope to purge tasks in specific repos on the basis of states. We can create another story for this and it should be easy to do then.